euphonic_sqw_models/euphonic_wrapper.py

```diff
@@ -51,14 +60,24 @@ class CoherentCrystal(object):
     # This a wrapper around the Euphonic ForceConstants and QpointPhononModes classes to make it easier to access from Matlab
     # It is meant to be used with a Matlab python_wrapper class and implements a horace_sqw function for use with Horace
 
-    defaults = {'debye_waller': None, 'debye_waller_grid': None, 'temperature': 0.0 * ureg('K'), 'bose': True,
-                'negative_e': False, 'conversion_mat': None, 'chunk': 5000, 'lim': np.inf, 'scattering_lengths': 'Sears1992',
-                'weights': None, 'asr': None, 'dipole': True, 'eta_scale': 1.0, 'splitting': True, 'insert_gamma': False, 
-                'reduce_qpts': True, 'use_c': None, 'n_threads': None, 'verbose': True}
+    defaults = {'debye_waller': None, 'debye_waller_grid': None,
+                'temperature': 0.0 * ureg('K'), 'bose': True,
+                'negative_e': False, 'conversion_mat': None,
+                'chunk': 5000, 'lim': np.inf,
+                'scattering_lengths': 'Sears1992', 'weights': None,
+                'asr': None, 'dipole': True,
+                'dipole_parameter': 1.0 ,'eta_scale': 1.0,
+                'splitting': True, 'insert_gamma': False,
+                'reduce_qpts': True, 'use_c': None,
+                'n_threads': None, 'verbose': True}
```

Comment:
The formatting here looks as if it only went half-way. Alignment of the second default on each line to make a nice column would also make it easier to read and determine, say, that ureg('K') went with temperature and not with bose.

Reply:
I don't think I've ever seen that kind of alignment in Python, I can't see anything specifically forbidding it in PEP8 but I can see that it discourages similar things (e.g. last part of [this section](https://www.python.org/dev/peps/pep-0008/#pet-peeves)). I know the rest of the file doesn't follow PEP8 but I'm hoping to fix that at some point...

What I can do though is put one dictionary entry on each line, that will probably make it a lot easier to read.